core/processor/abci.go

```diff
@@ -739,6 +739,7 @@ func (app *App) OnEndBlock(req tmtypes.RequestEndBlock) (ctx context.Context, re
 			AppVersion: AppVersion,
 		},
 	}
+	app.exec.BlockEnd(app.blockCtx)
```

Comment:
Not any issue I can't think of just now, but could there be any problems with calling that last, seeing that other engines are being call for the BlockEnd before it? Does order could matter here?

Reply:
yeah, I'm not sure about this one either. I just added it to the bottom to not interfere with the existing calls